data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -79,6 +87,9 @@ protected JacksonEvent(final Builder builder) {
         }
 
         this.jsonNode = getInitialJsonNode(builder.data);
+
+        zonedDateTime = ZonedDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXX"));
```

Comment:
I want to point out that I used `DateTimeFormatter` here because if we don't have it trailing zeros will be removed from timestamp as seen in below example. And it will return `String` type instead of `ZonedDateTime` object.
```
expected :{\"event_timestamp\":\"2022-01-26T13:35:32.270-06:00\"}
Actual   :{\"event_timestamp\":\"2022-01-26T13:35:32.27-06:00\"}
```